app/src/renderer/components/governance/PageGovernance.vue

```diff
@@ -67,14 +57,14 @@ export default {
     showModalPropose: false
   }),
   computed: {
-    ...mapGetters([`proposals`, `filters`, `depositDenom`, `connected`])
+    ...mapGetters([`proposals`, `filters`, `depositDenom`, `connected`]),
+    proposalList() {
+      return Object.keys(this.proposals.proposals)
```

Comment:
hmm, it'd be better to change the getter to just use `this.proposals` (separate issue probably)

Reply:
Yeah this was the renaming issue we were tackling some weeks ago, but I think we are reading tallies, loading and some other property